modules/gcp/centos-gfx/centos-gfx-startup.sh.tmpl

```diff
@@ -241,32 +240,32 @@ join_domain()
 
         # Wait for AD service account to be set up
         yum -y install openldap-clients
-        log "--> Wait for AD account ${ad_service_account_username}@${domain_name} to be available"
+        log "--> Waiting for AD account ${ad_service_account_username}@${domain_name} to be available..."
         until ldapwhoami -H ldap://${domain_controller_ip} -D ${ad_service_account_username}@${domain_name} -w $AD_SERVICE_ACCOUNT_PASSWORD -o nettimeout=1 > /dev/null 2>&1
         do
-            log "${ad_service_account_username}@${domain_name} not available yet, retrying in 10 seconds..."
+            log "--> ${ad_service_account_username}@${domain_name} not available yet, retrying in 10 seconds..."
             sleep 10
         done
 
         # Join domain
-        log "--> Install required packages to join domain"
+        log "--> Installing required packages to join domain..."
         yum -y install sssd realmd oddjob oddjob-mkhomedir adcli samba-common samba-common-tools krb5-workstation openldap-clients policycoreutils-python
 
-        log "--> Restarting messagebus service"
+        log "--> Restarting messagebus service..."
         if ! (systemctl restart messagebus)
         then
-            log "--> Failed to restart messagebus service"
+            log "--> ERROR: Failed to restart messagebus service."
             return 106
         fi
 
-        log "--> Enable and start sssd service"
+        log "--> Enabling and starting sssd service..."
         if ! (systemctl enable sssd --now)
         then
-            log "Failed to start sssd service"
+            log "--> ERROR: Failed to start sssd service."
             return 106
         fi
 
-        log "--> Joining the domain"
+        log "--> Joining the domain..."
         if [[ -n "$OU" ]]
```

Comment:
Seems like $OU is never defined. could double compare with the CAM script and maybe get rid of this in the future.